queries.yaml

```diff
@@ -112,6 +112,7 @@ layers:
     transform:
       - TileStache.Goodies.VecTiles.transform.tags_create_dict
       - TileStache.Goodies.VecTiles.transform.add_iata_code_to_airports
+      - TileStache.Goodies.VecTiles.transform.normalize_leisure_kind
```

Comment:
If we're normalizing them all to `fitness` should this be `normalize_fitness_kind` instead?